crates/devices/virtio-blk/src/lib.rs

```diff
@@ -1,11 +1,15 @@
-// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
-//
-// SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+//! Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//!
+//! SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
```

Comment:
We shouldn't mark as documentation the copyright license. Instead, we should add a summary of the crate, e.g.

```rust
//! Provides abstractions for the virtio block device.
```
An example [here](https://github.com/rust-vmm/vm-virtio/blob/master/src/lib.rs) or [here](https://github.com/rust-vmm/vm-memory/blob/master/src/lib.rs).

Reply:
Oops, thats silly from me. Thanks for finding it!